runtime/compiler/control/rossa.cpp

```diff
@@ -1425,6 +1425,9 @@ onLoadInternal(
    codeCacheConfig._maxNumberOfCodeCaches = maxNumberOfCodeCaches;
    codeCacheConfig._canChangeNumCodeCaches = (TR::Options::getCmdLineOptions()->getNumCodeCachesToCreateAtStartup() <= 0);
 
+   UDATA totalCacheBytes = codeCacheConfig._codeCacheKB >> 10;
```

Comment:
May bad, here. We need to shift left to multiply by 1024.

Reply:
Another thing: we should avoid overflow by  grouping `totalCacheBytes / 100`, to be performed first (even if we lose some bytes due to truncation). Otherwise `_highCodeCacheOccupancyPercentage * totalCacheBytes` could overflow 32-bit (on 32-bit platforms) and result in very low values